tests/functional/modules/test_zos_copy_func.py

```diff
@@ -2433,79 +2420,16 @@ def test_copy_ps_to_existing_uss_dir(ansible_zos_module):
 @pytest.mark.seq
 def test_copy_ps_to_non_existing_ps(ansible_zos_module):
     hosts = ansible_zos_module
-    src_ds = TEST_PS
+    src_ds = get_tmp_ds_name()
     dest = get_tmp_ds_name()
 
     try:
-        hosts.all.zos_data_set(name=dest, state="absent")
+        hosts.all.shell(cmd=f"decho 'this is a test line' '{src_ds}'")
         copy_res = hosts.all.zos_copy(src=src_ds, dest=dest, remote_src=True)
         verify_copy = hosts.all.shell(
             cmd="cat \"//'{0}'\"".format(dest), executable=SHELL_EXECUTABLE
         )
 
-        # Copying the remote loadlibs in USS to a local dir.
```

Comment:
Do we need to test this flow on a new test or can we just leave it out now? I remember this came about from a support issue and it was a bit contentious at the time, I could be wrong though

Reply:
oh it is tested in test_copy_local_pds_loadlib_to_pds_loadlib but this looks like a merge error, so I removed it.